cli/src/generate/render.rs

```diff
@@ -263,7 +263,6 @@ impl Generator {
 
     fn add_pragmas(&mut self) {
         add_line!(self, "#if defined(__GNUC__) || defined(__clang__)");
-        add_line!(self, "#pragma GCC diagnostic push");
```

Comment:
why this change?

Reply:
No point in pushing if it's not popped. It will apply till the end of the file anyway.

Reply:
oh ok, sounds good.